system/system.go

```diff
@@ -51,11 +56,83 @@ func (i *Info) Clone() *Info {
 		MessagesDropped:     atomic.LoadInt64(&i.MessagesDropped),
 		Retained:            atomic.LoadInt64(&i.Retained),
 		Inflight:            atomic.LoadInt64(&i.Inflight),
-		InflightDropped:     atomic.LoadInt64(&i.InflightDropped),
-		Subscriptions:       atomic.LoadInt64(&i.Subscriptions),
-		PacketsReceived:     atomic.LoadInt64(&i.PacketsReceived),
-		PacketsSent:         atomic.LoadInt64(&i.PacketsSent),
-		MemoryAlloc:         atomic.LoadInt64(&i.MemoryAlloc),
-		Threads:             atomic.LoadInt64(&i.Threads),
+		//		InflightDropped:     atomic.LoadInt64(&i.InflightDropped),
```

Comment:
Is this supposed to be commented?

Reply:
I forgot to mention, it appears InflightDropped is not used any more, I can't find any reference where the broker updates it.